flow/project.py

```diff
@@ -3629,9 +3895,9 @@ def _main_status(self, args):
 
         start = time.time()
         try:
-            self.print_status(jobs=jobs, **args)
+            self.print_status(jobs=aggregates, **args)
         except NoSchedulerError:
-            self.print_status(jobs=jobs, **args)
+            self.print_status(jobs=aggregates, **args)
```

Comment:
This doesn't make sense, it'll just throw the same error again. I tried looking back in the history and this code has been here a very long time. It was introduced wayyyy back in [59e2df3cf413261deb999cffebd4c434a7ad2139](https://github.com/glotzerlab/signac-flow/commit/59e2df3cf413261deb999cffebd4c434a7ad2139) back when we still passed scheduler classes around. This block can be removed here.

Reply:
@bdice removed this block in a commit, so I'm going to resolve this conversation.